src/meta/src/hummock/manager/mod.rs

```diff
@@ -1189,6 +1188,22 @@ where
 
                 trigger_version_stat(&self.metrics, current_version, &versioning.version_stats);
 
+                {
+                    let group_config = self
+                        .get_compaction_group_config(compact_task.compaction_group_id)
+                        .await;
+
+                    trigger_lsm_stat(
```

Comment:
`trigger_lsm_stat` is executed when holding the version and compaction lock and we will basically traverse the LSM tree to calculate the pending bytes and compression ratio. Can this be a problem?